jiant/tasks/lib/mnli.py

```diff
@@ -18,31 +18,31 @@
 @dataclass
 class Example(BaseExample):
     guid: str
-    input_premise: str
-    input_hypothesis: str
+    premise: str
+    hypothesis: str
```

Comment:
This change was required to reflect the exact field names in the mnli dataset. Although not strictly necessary, it simplifies the test format.

Reply:
Cool, looks there'll be opportunities to do the same in `qnli`, `rte`, `snli` and `wnli`, `anli`, `commitmentbank`, and `scitail`.